### PR TITLE
[Vue] migrate PiwikMarketplaceLicenseController controller and twig template to Vue

### PR DESCRIPTION
**Pull request by diosmosis**
### Description:

Changes:
* Migrate PiwikMarketplaceLicenseController controller and twig template to vue.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by sgiehl** [comment]
@diosmosis I locally tried to merge in the latest changes from 4.x-dev before testing. But that resulted in an error when opening the Marketplace:
image
Could you please check that?

File: plugins/Marketplace/templates/licenseform.twig [hunk 1]
```diff
@@ -1,75 +1,10 @@
...
-
-                <a href="{{ linkTo({'action': 'subscriptionOverview'}) }}" class="btn valign">
-                    {{ 'Marketplace_ViewSubscriptions'|translate }}
-                </a>
-
-                {% if isAutoUpdatePossible and isPluginsAdminEnabled and paidPluginsToInstallAtOnce|length %}
-                    <a href="javascript:;" class="btn installAllPaidPlugins valign">
-                        {{ 'Marketplace_InstallPurchasedPlugins'|translate }}
-                    </a>
-                    {% include '@Marketplace/paid-plugins-install-list.twig' %}
```
Comment (sgiehl) [inline plugins/Marketplace/templates/licenseform.twig:42]: That files doesn't seem to be removed. Is it still in use somewhere?

**Review by sgiehl — CHANGES_REQUESTED** [review]
By now I have only done some code review. Once the merge issue was fixed some local testing needs to be done.

**Comment by diosmosis** [comment]
@sgiehl fixed

File: plugins/Marketplace/vue/src/LicenseKey/LicenseKey.vue [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+                <ul>
+                  <li v-for="pluginName in paidPluginsToInstallAtOnce" :key="pluginName">
+                    {{ pluginName }}
+                  </li>
+                </ul>
+                <p>
+                  <input
+                    role="install"
+                    type="button"
+                    :href="installAllPaidPluginsLink"
```
Comment (sgiehl) [inline plugins/Marketplace/vue/src/LicenseKey/LicenseKey.vue]: This needs to be a `data-href` attribute in order to work:
See https://github.com/matomo-org/matomo/blob/ee41a13e975d5ffeae278a78b786c8326ead694b/plugins/Morpheus/javascripts/piwikHelper.js#L412-L416

Comment (diosmosis) [reply plugins/Marketplace/vue/src/LicenseKey/LicenseKey.vue]: :+1: fixed

**Review by sgiehl — APPROVED** [review]

**Pull request merged**